languages/go/concepts/methods/about.md

```diff
@@ -1 +1,59 @@
-TODO: add information on methods concept
+A [method][methods] is a function with a special _receiver_ argument. The receiver appears in its own argument list between `func` keyword and the name of the method.
+
+```go
+func (receiver type) MethodName(parameters) (returnTypes){
+
+}
+```
+
+You can only define a method with a receiver whose type is defined in the same package as the method. The receiver can be either a struct type or a non-struct type as in the example below.
+
+```go
+package main
+
+import "fmt"
+
+type Name string
+
+func (s Name) Greetings() string {
+	return fmt.Sprintf("Welcome %s !", s)
+}
+
+func main() {
+	s := Name("Bronson")
+	fmt.Println(s.Greetings())
+}
+// Output: Welcome Bronson !
```

Comment:
Maybe remove all but the type declaration and the method? Could make it easier to understand.

Reply:
Sure, I've removed them.